content/en/real_user_monitoring/guide/best-practices-tracing-native-ios-android-apps.md

```diff
@@ -0,0 +1,157 @@
+
+### Sampling parameters
+
+The following sampling rate parameters control different aspects of data collection:
+
+{{< tabs >}}
+{{% tab "iOS" %}}
+
+| Parameter                                   | Description                                                      |
+|---------------------------------------------|------------------------------------------------------------------|
+| `Trace.sampleRate`                          | Controls the percentage of manually instrumented spans collected by the tracer.|
+| `urlSessionTracking.firstPartyHostsTracing.sampleRate` | Controls the percentage of network requests traced for first-party hosts. |
+| `RUM.sessionSampleRate`                     | Controls the percentage of user sessions that are recorded for RUM.|
+| `WebViewTracking.logsSampleRate`            | Controls the percentage of logs collected from WebViews.          |
+
+{{% /tab %}}
+{{% tab "Android" %}}
+
+| Parameter                                   | Description                                                      |
+|---------------------------------------------|------------------------------------------------------------------|
+| `Trace.sampleRate`                          | Controls the percentage of manually instrumented spans collected by the tracer.|
+| `urlSessionTracking.firstPartyHostsTracing.sampleRate` | Controls the percentage of network requests traced for first-party hosts. |
+| `RUM.sessionSampleRate`                     | Controls the percentage of user sessions that are recorded for RUM.|
+| `WebViewTracking.logsSampleRate`            | Controls the percentage of logs collected from WebViews.          |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### How sampling works
+
+Sampling affects different types of spans you create in your mobile app:
+
+- **Local span sampling** applies to manually instrumented spans (like business spans or performance profiling spans). It's controlled by the `Trace.sampleRate` parameter. For example, if you set this rate to 50, all manually created spans are sent to Datadog, but only 50% are visible in the UI. Each span event includes the `_dd.agent_psr` field (the sampling rate) and `metrics._sampling_priority_v1` (1 for sampled, 0 for not sampled).
+
+- **Distributed trace sampling** applies to traces that cross service boundaries, such as network requests to your backend (relevant for the "Wrap a frontend-to-backend distributed trace" use case). This is controlled by the `urlSessionTracking.firstPartyHostsTracing.sampleRate` parameter. If set to 50, only half of backend requests have the sampled flag set to true, as indicated by the [W3C trace context][7]. All Datadog agents honor this decision, so you see 50% of distributed traces in the UI.
```

Comment:
```suggestion
- **Distributed trace sampling** applies to traces that cross service boundaries, such as network requests to your backend (relevant for the "Wrap a frontend-to-backend distributed trace" use case). This is controlled by the `urlSessionTracking.firstPartyHostsTracing.sampleRate` parameter for iOS and `DatadogInterceptor.Builder.setTraceSampler` parameter for Android. If set to 50, only half of backend requests have the sampled flag set to true, as indicated by the [W3C trace context][7]. All Datadog agents honor this decision, so you see 50% of distributed traces in the UI.
```

Reply:
Should we also link other platforms here such as RN, Flutter, KMP and Unity?
Or just redirect to these set of pages for a deeper read: https://docs.datadoghq.com/real_user_monitoring/correlate_with_other_telemetry/apm/